astroplan/tests/test_scheduler.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import datetime as dt
+import pytest
+
+import numpy as np
+import astropy.units as u
+from astropy.time import Time
+from astropy.coordinates import SkyCoord, get_sun
+from astropy.utils import minversion
+
+from ..moon import get_moon
+from ..observer import Observer
+from ..target import FixedTarget
+from ..constraints import (AltitudeConstraint, AirmassConstraint, AtNightConstraint)
```

Comment:
Generally it's good practice to install/enable a linter for you editor of choice. It will highlight unused imports, unused variables, PEP8 errors, etc.

 If you happen to use emacs then there is a rather good guide in the astropy dev docs, but similar solutions can be found for other editors, too.